### PR TITLE
[config] Remove _get_breakout_cfg_file_name helper function

### PR DESCRIPTION
**Pull request by vdahiya12**
this PR addresses replacing _get_breakout_cfg_file_name
function with device_info.get_path_to_port_config_file().
This is part of a larger clean up of functions which return
the port_config_file which reside outside of
sonic_py_common packages. This is motivated due to
redundancy of code as well as making it easier to change
the default behaviour of helper utility function
port_config_file unified and in one place.

Signed-off-by: vaibhav-dahiya <vdahiya@microsoft.com>

### CODE REVIEW

File: config/main.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 import threading
 import time
 
-from minigraph import parse_device_desc_xml
+from minigraph import earse_device_desc_xml
```
Comment (jleveque) [inline config/main.py:15]: Looks like an accidental change.

File: config/main.py [hunk 2]
```diff
@@ -12,7 +12,7 @@
...
 from portconfig import get_child_ports, get_port_config_file_name
```
Comment (jleveque) [inline config/main.py:16]: It appears that `get_port_config_file_name` is no longer needed now, either.

File: config/main.py (hunk 1 above)
Comment (vdahiya12) [reply config/main.py]: fixed

File: config/main.py (hunk 2 above)
Comment (vdahiya12) [reply config/main.py]: fixed

**Comment by jleveque** [comment]
@samaity: Please review.

**Comment by samaity** [comment]
@jleveque @vdahiya12 , LGTM!

**Pull request merged**